### DISCUSSION TITLE
Tooltip on Click

### DISCUSSION THREAD

**Discussion by cubecleveland** [radix-ui/primitives#1500] [Help]
Hello, is it possible to show a tooltip on click or do we need to use a popover ?

**Comment by benoitgrelard** [comment]
Hi @cubecleveland, showing a tooltip on click does not make sense because a tooltip is a "tip" on a "tool" to explain its action, so the the tool (most of the time a button) will already have its own action on click.

So yes, more than likely you are looking for `Popover` here.
